### PR TITLE
Further documentation

### PR DESCRIPTION
**Pull request by andrii-balitskyi**

### CODE REVIEW

File: seamapi/action_attempts.py [hunk 1]
```diff
@@ -17,14 +17,57 @@ def to_action_attempt_id(action_attempt: Union[ActionAttemptId, ActionAttempt])
...
 class ActionAttempts(AbstractActionAttempts):
+    """
+    A class used to retreive action attempt data
+    through interaction with Seam API
+
+    ...
+
+    Attributes
+    ----------
+    seam : dict
```
Comment (andrii-balitskyi) [inline seamapi/action_attempts.py]: class should be of type type not dict?

File: seamapi/access_codes.py [hunk 2]
```diff
@@ -101,7 +102,7 @@ def get(self, access_code: Union[AccessCodeId, AccessCode]) -> AccessCode:
```
Comment (andrii-balitskyi) [inline seamapi/access_codes.py:97]: access_codes/get also accepts `device_id`, should we add it?

File: seamapi/action_attempts.py [hunk 3]
```diff
@@ -8,23 +8,61 @@
...
+from seamapi.utils.convert_to_id import to_action_attempt_id
 
 
-def to_action_attempt_id(action_attempt: Union[ActionAttemptId, ActionAttempt]) -> str:
-    if isinstance(action_attempt, str):
-        return action_attempt
-    return action_attempt.action_attempt_id
+class ActionAttempts(AbstractActionAttempts):
+    """
+    A class used to retreive action attempt data
```
Comment (seveibar) [inline seamapi/action_attempts.py]:
```suggestion
    A class used to retrieve action attempt data
```

File: seamapi/action_attempts.py [hunk 4]
```diff
@@ -8,23 +8,61 @@
...
 
-class ActionAttempts(AbstractActionAttempts):
     seam: Seam
 
     def __init__(self, seam: Seam):
+        """
+        Parameters
+        ----------
+        seam : Seam
+          Intial seam class
```
Comment (seveibar) [inline seamapi/action_attempts.py]:
```suggestion
          Initial seam class
```

File: seamapi/connect_webviews.py [hunk 5]
```diff
@@ -3,18 +3,62 @@
...
 )
 import requests
-from typing import List, Optional
+from typing import List, Optional, Union
+from seamapi.utils.convert_to_id import to_connect_webview_id
 
 
 class ConnectWebviews(AbstractConnectWebviews):
+    """
+    A class used to retreive connect webview data
```
Comment (seveibar) [inline seamapi/connect_webviews.py]:
```suggestion
    A class used to retrieve connect webview data
```

File: seamapi/connected_accounts.py [hunk 6]
```diff
@@ -2,18 +2,59 @@
...
+
 import requests
-from typing import List, Optional
+from typing import List, Optional, Union
+from seamapi.utils.convert_to_id import to_connected_account_id
 
 
 class ConnectedAccounts(AbstractConnectedAccounts):
+    """
+    A class used to retreive connected account data
```
Comment (seveibar) [inline seamapi/connected_accounts.py]:
```suggestion
    A class used to retrieve connected account data
```

File: seamapi/connected_accounts.py [hunk 7]
```diff
@@ -2,18 +2,59 @@
...
+    """
+
     seam: Seam
 
     def __init__(self, seam: Seam):
+        """
+        Parameters
+        ----------
+        seam : Seam
+          Intial seam class
```
Comment (seveibar) [inline seamapi/connected_accounts.py]:
```suggestion
          Initial seam class
```

**Review by seveibar — APPROVED** [review]

**Review by seveibar — APPROVED** [review]

**Pull request merged**